internal/server/data/data.go

```diff
@@ -390,3 +405,35 @@ func InfraConnectorIdentity(db ReadTxn) *models.Identity {
 	}
 	return connector
 }
+
+const slowQueryThreshold = time.Second
+
+func logQuery(query string, err error, startedAt time.Time, rows int64) {
+	level := zerolog.TraceLevel
+
+	elapsed := time.Since(startedAt)
+	switch {
+	case errors.Is(err, sql.ErrNoRows), errors.Is(err, gorm.ErrRecordNotFound):
+		level = zerolog.WarnLevel
+	case elapsed > slowQueryThreshold:
+		level = zerolog.WarnLevel
+	}
+
+	query = normalizeQueryString(query)
```

Comment:
Gorm was normalizing the query, I guess because it was parsing the full query and rewriting it for the appropriate dialect.  Since that doesn't happen anymore here I've done some basic normalization to remove extra whitespace.